Resources/views/menu.html.twig

```diff
@@ -0,0 +1,3 @@
+{% extends 'knp_menu.html.twig' %}
+
+{% block label %}{{ parent()|trans({}, item.getExtra('translation_domain', 'messages')) }}{% endblock %}
```

Comment:
this is wrong:

- It breaks the support of safe labels available in the library, because it will always escape the return value of the trans tag
- it will double-escape the existing label (in case the translator does not change the string) because it is already escaped in the parent
- the translator might fail to translate if the escaping in the parent changes the string as it won't match the translation key anymore.